### ISSUE TITLE
[Feature] More columns

### ISSUE THREAD

**Issue by frankhaugen** [aptakube/aptakube#343]
### What problem are you trying to solve? Do you a suggestion on how to solve it?

Currently the Pod-view shows these columns:
![image](https://github.com/user-attachments/assets/867f549e-426f-4589-8b73-5e86ce8ec61b)

It would be really nice to have the "Requested CPU", "Requested Memory", "% Requested CPU", "% Request Memory", and same for limit. This goes for most "workloads"-view.

Namespace-view could also have more, like counts etc. its very little information right now:
![image](https://github.com/user-attachments/assets/e64e96a5-3d9f-4ee3-94bd-a33024ceab3d)

Love the product but I end up going into k9s to get some of this data on a daily basis.

**Comment by giuseppetrematerra** [comment]
Yes, definitely! Columns are customizable as now, adding those should not be a problem.
